### ISSUE TITLE
Dist::Zilla 6.x breaks [Author::Plicease::Test] maybe others

### ISSUE THREAD

**Issue by plicease** [plicease-legacy/Dist-Zilla-PluginBundle-Author-Plicease#13]
```
Path::Tiny paths require defined, positive-length parts at /home/travis/perl5/perlbrew/perls/5.20/lib/site_perl/5.20.0/Dist/Zilla/Role/MintingProfile/ShareDir.pm line 22.
```

**Comment by karenetheridge** [comment]
Please let me know what version this was fixed in and I'll update Dist::Zilla's `x_breaks` metadata accordingly!

**Comment by plicease** [comment]
Fixed in 2.03.  Thanks!
